### DISCUSSION TITLE
SimpleMeshLayer Properties Mesh

### DISCUSSION THREAD

**Discussion by floki2020** [visgl/deck.gl#6422] [Q&A]
Thanks for your help！
I wanna create a `SimpleMeshLayer`  , Refer to the example on the official website！
```
  registerLoaders([OBJLoader])
  const layer = new SimpleMeshLayer({
    id: _layer.id,
    mesh: '/buildings/Tank.obj',
    loaders: [OBJLoader]
  })
```
but it didn't work! I used the registerLoaders register the loader 
[ official demo](https://github.com/visgl/deck.gl/blob/master/examples/website/mesh/app.js)
 `mesh`  only support the type of `SimpleMesh`

What should i do？
Thanks again！

deck.GL :8.6
react ：17
typescript：4

**Comment by Pessimistress** [comment]
Please create a Code Sandbox if you need help debugging your own application.
